.circleci/config.yml

```
@@ -25,7 +25,7 @@ parameters:
     default: "solbuildpackpusher/solidity-buildpack-deps@sha256:092da5817bc032c91a806b4f73db2a1a31e5cc4c066d94d43eedd9f365df7154"
   evm-version:
     type: string
-    default: london
+    default: london\
```

Looks like an accidental change:
```suggestion
    default: london
```